spec.html

```diff
@@ -8156,12 +8156,22 @@
 
     <emu-clause id="sec-immutable-prototype-exotic-objects">
       <h1>Immutable Prototype Exotic Objects</h1>
-      <p>An <dfn>immutable prototype exotic object</dfn> is an exotic object that has an immutable [[Prototype]] internal slot.</p>
+      <p>An <dfn>immutable prototype exotic object</dfn> is an exotic object that has an [[Prototype]] internal slot that will not change once it is initialized.</p>
+
+      <p>Immutable prototype exotic objects have the same internal slots as ordinary objects. They are exotic only in the following internal methods. All other internal methods of immutable prototype exotic objects that are not explicitly defined below are instead defined as in <a href="#sec-ordinary-object-internal-methods-and-internal-slots">ordinary objects.</a></p>
 
       <emu-clause id="sec-immutable-prototype-exotic-objects-setprototypeof-v">
         <h1>[[SetPrototypeOf]] (_V_)</h1>
         <p>When the [[SetPrototypeOf]] internal method of an immutable prototype exotic object _O_ is called with argument _V_, the following steps are taken:</p>
         <emu-alg>
+          1. Return ? SetImmutablePrototype(O, V).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-set-immutable-prototype">
```

Comment:
Maybe leave a source file comment `<!-- This is used by host environment specs, even though it only has one consumer in this spec; don't inline it -->`